.env.sample

```diff
@@ -1,2 +1,9 @@
+# Node environment: 'development' | 'production'
 NODE_ENV=development
+
+# Port for Express server
+PORT=5050
+
+# MongoDB connection URI
 MONGO_URI=mongodb://your-username:your-password@your-cluster.mongodb.net/your-database-name?retryWrites=true&w=majority
```

Comment:
Some of the comments in the code seem unnecessary or redundant. 

For example, commenting like this:
```js
// A boolean variable that checks if something is active or not
const isActive = true; 
```

Will not add anything, because its super clear from the variable name what this is. 

Same goes for `PORT` `MONGO_URI`. 
The `NODE_ENV` is fine because you want to explain what types are available - **but** the `Node environment` is not needed (we can clearly see that from the variable name)